config/metricConfigs/workspaceMetrics.ts

```diff
@@ -100,6 +100,27 @@ const allowedMetrics: MetricsConfig = [
     ],
     protocol: "statsd"
   },
+  {
+    name: "workspace_time_to_load_dashboard",
+    help: "measures the time to render completely the dashboard",
+    type: "distribution",
+    labels: [],
+    protocol: "statsd"
+  },
+  {
+    name: "workspace_time_to_load_publication",
+    help: "measures the time to render completely a publication",
+    type: "distribution",
+    labels: [],
+    protocol: "statsd"
+  },
+  {
+    name: "workspace_time_to_load_editor",
```

Comment:
Is this still necessary? Wasn't this already added by Jeroen?

Reply:
The one by Jeroen is `workspace_time_to_initial_content_render_start` I think